dist/node/mesh.js

```diff
@@ -26,29 +26,31 @@ class mesh {
     let cozPort = 8880
     let cozNetwork = 'test'
 
-    if (this.network === 'mainnet') {
-      neoPort = 10332
-      cozPort = 8080
-      cozNetwork = 'seed'
-    }
+    if (!this.nodes.length) {

```

Comment:
as mentioned in the previous comment. 'seeds' is probably a better term for the initial list being sent in to the system since those seeds may "eventually" respond with more 'nodes' to get populated.